java-ast/addon/src/main/java/org/jboss/windup/ast/java/data/TypeReferenceLocation.java

```diff
@@ -83,14 +83,16 @@
     VARIABLE_DECLARATION("Declares variable"),
 
     /**
-     * A Java class implements the specified type.
+     * A Java class implements the specified type; works transitively.
      */
     IMPLEMENTS_TYPE("Implements type"),
 
     /**
      * A variable initalization expression value. For example, this would resolve to "mypackage.MyConstants.FOO" for the following field declaration:
      *
+     * <pre>
      * int foo = mypackage.MyConstants.FOO;
+     * </pre>
      */
     VARIABLE_INITIALIZER("Variable Initializer");
 
```

Comment:
I would also somehow provide examples and this comment: " TYPE is registered as soon as a java class is found, transitively on superclasses. Then Implements/EXTENDS are registered transitively on both, interfaces/classes and also all the method_calls are registered transitively but only transitively on the interfaces (not extensions)"